reference/configuration/tidb-server/configuration-file.md

```diff
@@ -252,7 +252,7 @@ Configuration items related to performance.
 
 - The maximum number of statements allowed in a single TiDB transaction.
 - Default value: `5000`
-- If a transaction does not roll back or commit after the number of statements exceeds `stmt-count-limit`, TiDB returns the `statement count 5001 exceeds the transaction limitation, autocommit = false` error.
+- If a transaction does not roll back or commit after the number of statements exceeds `stmt-count-limit`, TiDB returns the `statement count 5001 exceeds the transaction limitation, autocommit = false` error. This configuration takes effect **only** in the retriable optimistic transaction. If you use the pessimistic transaction or have disabled the transaction retry, the number of statements in a transaction is not limited by this configuration.
```

Comment:
Should we add the link of the transaction retry?

Reply:
The transaction retry part is to be translated in [#1695](https://github.com/pingcap/docs/pull/1695), so I remove the link here. When #1695 is merged, I'll add this link here.

Reply:
got